stories/chats/src/BubbleQRCode.tsx

```diff
@@ -119,15 +130,18 @@ export const BubbleQRCode: React.FC<QRCodeMessage> = (message: QRCodeMessage) =>
                     roundedCornerStyle,
                 ]}
             >
-                <View style={styles.qrCode}>
-                    <UIQRCodeView
-                        size={QRCodeSize.Medium}
-                        testID={`chat_qr_code_${data}`}
-                        type={QRCodeType.Square}
-                        logo={UIAssets.icons.brand.tonSymbolBlack}
-                        value={data}
-                    />
-                </View>
+                <UIPressableArea onPress={onPress}>
```

Comment:
How about to wrap the view with the pressable area only in case `message.onPress` is provided (as it's optional)?